app/controllers/api/v1/task/items/concerns/rendering.rb

```diff
@@ -18,8 +18,8 @@ def render_json_with_attributes(task, status)
       render_json_with_success(status:, data:)
     end
 
-    def map_json_attributes((id, name, completed_at, created_at, updated_at, task_list_id))
-      {id:, name:, completed_at:, created_at:, updated_at:, task_list_id:}
+    def map_json_attributes((id, name, completed, completed_at, created_at, updated_at, task_list_id))
+      {id:, name:, completed:, completed_at:, created_at:, updated_at:, task_list_id:}
```

Comment:
@lazaronixon, why is adding a new column necessary? The task item is incomplete if a completed_at is null.